test/test_misc.py

```diff
@@ -0,0 +1,17 @@
+# Owner(s): ["module: tests"]
```

Comment:
I would put this into test/test_torch.py, that is where a lot of our basic tensor tests go

Reply:
I would agree that a test_misc is not very descriptive. Also no team is signed up to look at module: tests so I think we should avoid using it here as much as possible.

Reply:
Drumroll (this is where I got the idea in the first place):
https://github.com/pytorch/pytorch/blob/ed15fa7cc2adf9918e716d669ec0ed77c488fc69/test/test_torch.py#L1


And this is why I've picked this one for new file, but sure, can move it to `test_torch.py`